Add AFN In ListOrders

mabuak opened the pull request:
Now, we can see the MFN and AFN in list orders together.
If user need to see MFN or AFN, just used 
$client->ListOrders($from, false, array(), 'AFN');

mabuak committed:
Add AFN In ListOrders

Now, we can see the MFN and AFN in list orders together.
If user need to see MFN or AFN, just used 
$client->ListOrders($from, false, array(), 'AFN');

meertensm commented:
Hi Mabuak,
Thank you for you efforts. You should not change the default value for function parameters. This is not backward-compatible. The other changes you made are good though, and i will merge them when you change the parameter value.

mabuak commented:
Hi @meertensm , that is example, user still can used like this

// Returns orders created or updated during a time frame that you specify.
$client->ListOrders($from, $allMarketplaces = false, $states = ['Unshipped', 'PartiallyShipped'], $FulfillmentChannel = ['MFN','AFN']);

mabuak committed:
Change Function Parameter Value to back

Change function parameter value to $FulfillmentChannel = 'MFN'.
And keep the the array for added the fullfillementchangel for MFN and AFN

mabuak committed:
Update MWSClient.php
